content/what-we-do/contents.lr

```diff
@@ -0,0 +1,130 @@
+
+
+#### text ####
+section_header: How we work with you
+----
+section_level: 1
+----
+markdown:
+
+1. We usually start with an introductory call to introduce ourselves and get an understanding of the unique challenges you and your company face. 
+2. From there we identify the simplest but most impactful project which we then want to understand in more detail. From there we propose a unique model architecture that will solve your particular problem. 
+3. If you already have a Bayesian model you are using we will review it and propose improvements for scalability and more efficient sampling.
+4. We usually require a data set although we have also developed models with mock data that the client can then fit themselves on their data.
+5. We then build the model in close collaboration with you and provide frequent updates on how the work is progressing. We pride ourselves on transparency, not just in our models but also in how we work.
+6. Our deliverables do not consist of fancy slide decks that solve theoretical problems but a packaged Python module with unit-tests, documentation, and training materials on how to understand and use the model to solve your problem.
+
+
+#### text ####
+section_header: Who we're here for
+----
+section_level: 1
+----
+markdown:
+
+If you meet one or more of these requirements, we are likely to add value to your business:
+
+* You want to learn something about your problem
+    * Machine learning models are usually "black box" and do not give insight on what they actually learned
+    * Bayesian models are "open box" and can be inquired into to understand exactly why certain answers are produced
+* You have structured data with any of these properties:
+    * Nested / hierarchical data
+    * Time-series data
+    * Other non-trivially structured data sets
+* Your problem is not a simple prediction problem
+    * If you have a large set of tabular data with labels and only care about prediction accuracy, machine learning will be the best tool
+    * If you care about more than prediction, Bayesian modeling will be a good fit
+* You make decisions with real-world consequences based on noisy data
+    * In this case it is important to take all possible interpretations of your data into account, not just the most likely
+    * Bayesian statistics naturally supports this by modeling beliefs as distributions rather than scalar values
+    * This results in more robust decisions that work across a whole range of plausible assumptions
+
+Problems in many domains meet these criteria as demonstrated by our customers in diverse fields ranging from finance, biotech, agriculture, pharma, adtech and more.
+
+If you are interested in learning more, please email us at [thomas.wiecki@pymc-labs.io](mailto:thomas.wiecki@pymc-labs.io). We look forward to hearing from you.
```

Comment:
We should also update https://github.com/pymc-labs/pymc-labs.github.io/blob/master/databags/contact.json to show the pymc-labs mail on the home page.

Reply:
Done.